pkg/registry/controller/rest.go

```diff
@@ -47,14 +48,26 @@ func (rcStrategy) NamespaceScoped() bool {
 func (rcStrategy) PrepareForCreate(obj runtime.Object) {
 	controller := obj.(*api.ReplicationController)
 	controller.Status = api.ReplicationControllerStatus{}
+	controller.Spec.SequenceNumber = 0
+	controller.Spec.SequenceNumber = 0
```

Comment:
Why allow it? I think people might be tempted to update if we allow initialization (the only use case i can think of is cloning all the keys by getting and reputting to a diffferent apiserver). I'm fine defaulting the SequenceNumber to the answer to life the universe and everything, if that's what you meant :)